Dockerfile

```diff
@@ -49,7 +47,8 @@ ENV PERL5LIB=$MINC_LIB_DIR/perl5/5.8.5 \
 RUN echo "cHJpbnRmICJrcnp5c3p0b2YuZ29yZ29sZXdza2lAZ21haWwuY29tXG41MTcyXG4gKkN2dW12RVYzelRmZ1xuRlM1Si8yYzFhZ2c0RVxuIiA+IC9vcHQvZnJlZXN1cmZlci9saWNlbnNlLnR4dAo=" | base64 -d | sh
 
 # Installing Neurodebian packages (FSL, AFNI, git)
-RUN apt-get install -y --no-install-recommends \
+RUN apt-get update && \
```

Comment:
These add slightly to build time, but if `apt-get install` is called too long after `apt-get update` (such as in a partial image rebuild), it can fail.